src/components/SetStateDialog.vue

```diff
@@ -52,103 +52,84 @@ export default {
           <span>Change the state of this {{ dialogType }}</span>
         </v-tooltip>
       </template>
-      <v-card :loading="markAsLoading">
-        <v-card-title>
-          Change the state of {{ taskRun ? taskRun.name : flowRun.name }}
-        </v-card-title>
-        <v-card-text>
-          <v-stepper v-model="e1">
-            <v-stepper-header>
-              <v-stepper-step :complete="e1 > 1" editable step="1">
-                Select State
-              </v-stepper-step>
-              <v-divider></v-divider>
-              <v-stepper-step :complete="e1 > 2" step="2">
-                Confirm
-              </v-stepper-step>
-            </v-stepper-header>
-            <v-divider></v-divider>
-            <v-stepper-items>
-              <v-stepper-content step="1">
-                <v-card-text>
-                  <div>
-                    <v-autocomplete
-                      v-model="selectedState"
-                      label="Select A State"
-                      :items="filteredStates"
-                      autocomplete="new-password"
-                      required
-                    >
-                    </v-autocomplete>
-                  </div>
-                  <v-checkbox
-                    v-if="flowRun"
-                    v-model="allTasks"
-                    class="pb-0 mb-0"
-                    label="Include non-mapped task runs"
-                  ></v-checkbox>
-                  <v-checkbox
-                    v-if="flowRun"
-                    v-model="childTasks"
-                    :disabled="!allTasks"
-                    class="pt-0 mt-0"
-                    label="Include mapped task runs - this may take some time"
-                  ></v-checkbox>
-                  <v-form ref="form" @submit.prevent>
-                    <v-text-field
-                      v-model="reason"
-                      autocomplete="off"
-                      label="Optional - Why are you changing state?"
-                      @keyup.enter="checkContinue()"
-                    ></v-text-field>
-                  </v-form>
-                </v-card-text>
-                <v-card-actions>
-                  <v-spacer />
-                  <v-btn
-                    v-disable-read-only-user="!selectedState"
-                    color="primary"
-                    @click="checkContinue"
-                  >
-                    Continue
-                  </v-btn>
 
-                  <v-btn text @click="reset">Cancel</v-btn>
-                </v-card-actions>
-              </v-stepper-content>
+      <v-card flat :loading="markAsLoading">
+        <div style="padding: 20px;">
+          <v-card-title class="headline word-break-normal mb-3" primary-title>
+            Change the state of {{ taskRun ? taskRun.task.name : flowRun.name }}
```

Comment:
Let's differentiate the task run/flow run name here. Also I think this will read:
```
Change the state of
```
for task runs without names, so we should add a check for that and use the task name with something like "run of <<task name>>" if it's not available.

Reply:
okay so something like this? 

```
 Change the run of
            {{
              taskRun
                ? taskRun.task.name
                : taskRun && !taskRun.task.name
                ? taskRun.name
                : flowRun.name
            }}
```

Reply:
Hm I think this:
```
 Change the run of
            {{
              taskRun ? taskRun.name ? taskRun.name : taskRun.task.name : flowRun.name
            }}
```